report-api-faq.md

```diff
@@ -107,13 +107,21 @@ available in the settlement reports on
 or on
 [SFTP report (deprecated)](https://developer.vippsmobilepay.com/docs/settlements/sftp-report-service).
 
+Match the reference on the transfer from your bank statement with the `reference` in the Report API.
+When you retrieve a series of payments (made by users in the app) with
+[`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`](https://developer.vippsmobilepay.com/api/report/#tag/reportv2ledgers/paths/~1report~1v2~1ledgers~1%7BledgerId%7D~1%7Btopic%7D~1feed/get)
+in the Report API, each of those payments' `pspReference` is the `reference` for the bank payout.
```

Comment:
Don't have much time now, but this is wrong..

Reply:
Yes, I had a conflict and it looks like I mangled it. Will make a new commit. Thanks so far!

Reply:
Changed, hopefully correct now. 🤞